src/deriver/api.py

```diff
@@ -13,6 +13,11 @@
 from .fragment_index import frag_index
 from .mate import mate
 import os
+import numpy as np
```

Comment:
also, `numpy` random methods use a different seed than the `random` methods 

Try not to use them unless you need to, like if you are working with arrays or whatever